src/Uno.Extensions.Navigation.Generators/ForceBindingsUpdateGenTool_1.cs

```diff
@@ -61,8 +61,11 @@ private string Generate(INamedTypeSymbol model)
 			bases: updateInterface,
 			code: $@"
 				ValueTask IForceBindingsUpdate.ForceBindingsUpdateAsync()
-				{{	
-					this.Bindings.Update();
+				{{
+					if(this.Bindings is not null)
+					{{
+						this.Bindings.Update();
+					}}
```

Comment:
Maybe?
```suggestion
					this.Bindings?.Update();
```

Reply:
Is there a scenario where this is being called with this.Bindings is null?